src/app/services/sms/sms_count.server.model.ts

```diff
@@ -34,8 +34,20 @@ const VerificationSmsCountSchema = new Schema<IVerificationSmsCountSchema>({
       required: true,
     },
   },
+  isOnboardedAccount: {
+    type: Boolean,
+  },
 })
 
+VerificationSmsCountSchema.pre<IVerificationSmsCountSchema>(
+  'save',
+  function (next) {
+    const formTwilioId = process.env.TWILIO_MESSAGING_SERVICE_SID
```

Comment:
we should be using the variable in `config`, not directly on process.env

Reply:
This also allows for mocking during tests

Reply:
great point, i'll change accordingly!